node/src/main/scala/org/bitcoins/node/SpvNodeMain.scala

```diff
@@ -55,14 +55,21 @@ object SpvNodeMain extends App with BitcoinSLogger {
 
   val emptyBloom =
     BloomFilter(numElements = 1, falsePositiveRate = 1, flags = BloomUpdateAll)
-  val spvNodeF = SpvNode(peer, chainApi, emptyBloom).start()
+
+  val spvNodeF = chainApiF.flatMap { chainApi =>
+    SpvNode(peer, chainApi, emptyBloom).start()
+  }
 
   val getHeight: Runnable = new Runnable {
 
-    def run: Unit =
-      spvNodeF
-        .flatMap(_.chainApi.getBlockCount)
-        .foreach(count => logger.debug(s"SPV block height: $count"))
+    def run: Unit = {
+      for {
+        spvNode <- spvNodeF
+        count <- spvNode.chainApi.getBlockCount
+      } yield {
```

Comment:
I get a compiler warning here: It's because we `yield` something in a unit-returning function. This removed the warning:
```scala
   def run: Unit =
      for {
        spvNode <- spvNodeF
        count <- spvNode.chainApi.getBlockCount
      } logger.debug(s"SPV block height: $count")
```

Reply:
That being said this is a leftover from some debugging I did, could probably delete this file altogether (the same functionality is achieved by running the chain+node+wallet server